### ISSUE TITLE
DotNet.createJSObjectReference() TS Type

### ISSUE THREAD

**Issue by RaoulRSV** [OfficeDev/Office-Add-in-samples#530]
**Is your feature request related to a problem? Please describe.**
TS type for  DotNet.createJSObjectReference() function 

_A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
While developping  Blazor Addin with TS it is necessary to use the DotNet.createJSObjectReference() function but I couldn't find any TS type for it

**Describe the solution you'd like**
An upgrade of the NPM DotNet type library or a turnaroud to use this function in TS

**Comment by aafvstam** [comment]
@danroth27 Is DotNet.createJSObjectReference() typically Blazor or .NET? At least I haven't found the implementation in the DotNet Type Library. Who owns that within the .NET space, can you redirect the question?

**Comment by danroth27** [comment]
Yeah, this needs to be handled on the Blazor side. Adding @mkartakmsft.

**Comment by davidchesnut** [comment]
@mkArtakMSFT okay if I assign this to you? Can you please help? Thanks!

**Comment by davidchesnut** [comment]
@danroth27 is there any update on this from the Blazor side? Thanks!

**Comment by RaoulRSV** [comment]
@danroth27  : done
